src/control/server/ctl_storage_rpc.go

```diff
@@ -151,6 +159,112 @@ func (c *ControlService) scanScm(ctx context.Context, req *ctlpb.ScanScmReq) (*c
 	return newScanScmResp(c.getScmUsage(ssr))
 }
 
+// Return the number of target managing a given NVME controleur
+func (c *ControlService) getTargetCount(pciAddr string) (int, error) {
+	for _, engineCfg := range c.srvCfg.Engines {
+		targetCount := engineCfg.TargetCount
+		for _, tierCfg := range engineCfg.Storage.Tiers {
+			if tierCfg.IsBdev() {
+				for index := range tierCfg.Bdev.DeviceList {
+					if pciAddr == tierCfg.Bdev.DeviceList[index] {
+						msg := "NVME controler %s is used by %d targets"
+						c.log.Debugf(msg, pciAddr, targetCount)
+						return targetCount, nil
+					}
+				}
+			}
+		}
+	}
+
+	return 0, errors.Errorf("unknown PCI address %s", pciAddr)
```

Comment:
the error is misleading if there are no engine configs

Reply:
Going to change it.